client/dev-tools-promo/components/dev-tools-promo.tsx

```diff
@@ -0,0 +1,96 @@
+import { Card } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { Button } from '@wordpress/components';
+import { translate } from 'i18n-calypso';
+import CardHeading from 'calypso/components/card-heading';
+import { useSelector } from 'calypso/state';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import './style.scss';
+
+type PromoCardProps = {
+	title: string;
+	text: string;
+	href: string;
+};
+
+const DevToolsPromo = () => {
+	const siteId = useSelector( getSelectedSiteId );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) ) || '';
+
+	const PromoCard = ( { title, text, href }: PromoCardProps ) => (
+		<Card className="dev-tools-promo__card">
+			<CardHeading>{ title }</CardHeading>
+			<p>{ text }</p>
+			<a href={ href }>{ translate( 'Learn more' ) }</a>
+		</Card>
+	);
+
+	const upgradeLink = `https://wordpress.com/checkout/${ encodeURIComponent( siteSlug ) }/business`;
+	const pluginsLink = `https://wordpress.com/plugins/${ encodeURIComponent( siteSlug ) }`;
+	return (
+		<div className="dev-tools-promo">
+			<div className="dev-tools-promo__hero">
+				<h1> { translate( 'Unlock all developer tools' ) }</h1>
+				<p>
+					{ translate(
+						'Upgrade to the Creator plan or higher to get access to all developer tools'
+					) }
+				</p>
+				<Button variant="primary" className="dev-tools-promo__button" href={ upgradeLink }>
+					{ translate( 'Upgrade now' ) }
+				</Button>
+				<Button variant="secondary" className="dev-tools-promo__button" href={ pluginsLink }>
+					{ translate( 'Browse plugins' ) }
+				</Button>
+			</div>
+			<div className="dev-tools-promo__cards">
+				<PromoCard
+					title={ translate( 'Hosting Configuration' ) }
```

Comment:
[nit] The structure of these cards is the same, so it would be better to define an array and loop through it. But it's up to you 🙂

Reply:
ahh fair.